tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -77,4 +77,35 @@ var _ = Describe("odo devfile catalog command tests", func() {
 			helper.MatchAllInOutput(output, wantOutput)
 		})
 	})
+
+	Context("When executing catalog describe component with a component name with a single component", func() {
+		It("should only give information about one component", func() {
+			output := helper.CmdShouldPass("odo", "catalog", "describe", "component", "maven")
+			helper.MatchAllInOutput(output, []string{"generateName: maven-", "- command: java -jar target/*.jar"})
```

Comment:
+1

Reply:
This command now gives a message if there are no projects in the devfile